site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -203,46 +203,87 @@ image::installation-guides/cloud/deployments-manage.png[Manage deployment,width=
+Test the upgrade process on development or test servers first.
 
 [NOTE]
 ====
-The upgrading described in this section only updates the version of FusionAuth. 
+The upgrade process described in this section updates the version of FusionAuth. 
 
 If you are looking to modify other attributes of your deployment, such as the region it is running in, the deployment size, or the service tier, review the <<Modifying Deployments>> section.
 ====
 
-Perform an upgrade at a time that works for your users, your team and your applications by https://account.fusionauth.io/[logging into your account]. Record the time you began the upgrade.
+==== Upgrade Process
 
-Navigate to the [breadcrumb]#Deployments# tab. Manage the deployment, then choose the "Upgrade" option. Select the version you are upgrading to from the [field]#Version# dropdown.
+To upgrade a deployment from version `1.x-1` to version `1.x`:
 
-Confirm the upgrade:
+* Test `1.x` on your staging or development environment.
+* Select a time that work for your users, your team and your applications.
+* https://account.fusionauth.io/[Log into your account].
+* Record the time you began the upgrade.
+* Navigate to the [breadcrumb]#Deployments# tab. Manage the deployment, then choose the "Upgrade" option. Select `1.x` from the [field]#Version# dropdown.
+* Confirm you want to start the upgrade:
 
 image::installation-guides/cloud/deployments-upgrade-confirm.png[Upgrading a deployment,width=1200,role=bottom-cropped]
 
-After confirmation, the deployment will be in an "Upgrading" state until finished. You can monitor the upgrade by viewing the [breadcrumb]#Deployments# tab. If you need to programmatically monitor the upgrade, you can call the link:/docs/v1/tech/apis/system#retrieve-system-status[System Status API]; when it returns success, the deployment upgrade is complete.
+* After confirmation, the deployment will be in an "Upgrading" state until finished. Monitor the upgrade by viewing the [breadcrumb]#Deployments# tab. To programmatically monitor the upgrade, call the link:/docs/v1/tech/apis/system#retrieve-system-status[System Status API].
+* When the deployment is in the `Active` state in the user interface, or when the System Status API returns success, the deployment is upgraded to `1.x`.
+
+==== Limitations
 
 You cannot downgrade a FusionAuth Cloud deployment version.
 
-==== Upgrades with a High-Availability Plan
+Please test upgrades before performing them on your production system.
 
-Upgrades with this tier have less downtime.
+==== Rolling Back From a Problematic Upgrade
 
-If the release notes indicate a database migration, the upgrade process will result in a few minutes of outage. Otherwise there will be no downtime.
+If an upgrade is problematic for whatever reason, and you identify the issues within your database backup retention period, open a https://account.fusionauth.io/account/support/[support ticket].
```

Comment:
If an upgrade is problematic, and the offending issue is identified in the same timeframe as your database backup retention period, open a https://account.fusionauth.io/account/support/[support ticket].

Reply:
Maybe 

> Please note, support tickets will be addressed per your Support Edition SLA, with deference granted towards the urgency of the situation.